netg@DESKTOP-EBH0I8A:~/devnet$ python3
Python 3.10.12 (main, Jul 29 2024, 16:56:48) [GCC 11.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> 
>>> 
>>> sw_config_data = {
...     "intf": "ge0/1",
...     "desc": "connected to user port",
...     "speed": "1000",
...     "duplex": "full"    
... }
>>> 
>>> sw_config_data.items()
dict_items([('intf', 'ge0/1'), ('desc', 'connected to user port'), ('speed', '1000'), ('duplex', 'full')])
>>> 
>>> sw_config_data.keys()
dict_keys(['intf', 'desc', 'speed', 'duplex'])
>>> sw_config_data.values()
dict_values(['ge0/1', 'connected to user port', '1000', 'full'])
>>> 
>>> sw_config_data.items()
dict_items([('intf', 'ge0/1'), ('desc', 'connected to user port'), ('speed', '1000'), ('duplex', 'full')])
>>> 
>>> tup_1 = (2,3,4)
>>> 
>>> tup_1 = [(2,3,4), (5,6,7), (8,9,10) ]
>>> 
>>> for k, v, m in tup_1:
...     print(k, v, m)
... 
2 3 4
5 6 7
8 9 10
>>> 
>>> 
>>> 
>>> 
>>> ping = f"ping {ip}"
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'ip' is not defined. Did you mean: 'id'?
>>> 
>>> ip = "10.2.3.4"
>>> 
>>> ping = f"ping {ip}"
>>> ping
'ping 10.2.3.4'
>>> 